### PR TITLE
[tests-only]Adds UI tests for local external storage

### LINKED ISSUE #32849: tests for external storage
Add API tests that confirm that external storage is listed in the "All files" list and the "External storage" section for ordinary users.

- [x] files can be uploaded, downloaded, deleted, moved in/out of external storage
- [ ] the "External storage" section shows a filtered view with just the external storage folders

Note: this will need some thought about creating external storage within a scenario, and removing it at the end. Probably for the ordinary user tests, that is best done with the similar before/after scenario code that the API tests use.

### PR DESCRIPTION
**Pull request by SwikritiT**
## Description
This PR adds UI tests for local external storage. Scenario added
-  the external section shows a filtered view with just the external storage folders

## Related Issue

- Fixes: https://github.com/owncloud/core/issues/32849

## How Has This Been Tested?

- locally
- CI

## Types of changes

- [ ] Bug fix (non-breaking change which fixes an issue)
- [ ] New feature (non-breaking change which adds functionality)
- [ ] Database schema changes (next release will require increase of minor version instead of patch)
- [ ] Breaking change (fix or feature that would cause existing functionality to change)
- [ ] Technical debt
- [x] Tests only (no source changes)

## Checklist:

- [ ] Code changes
- [ ] Unit tests added
- [x] Acceptance tests added
- [ ] Documentation ticket raised: <link> 
- [ ] Changelog item, see [TEMPLATE](https://github.com/owncloud/core/blob/master/changelog/TEMPLATE)

### CODE REVIEW

**Comment by ownclouders** [comment]
:boom: Acceptance tests pipeline <strong>webUIFiles-chrome-mariadb10.2-php7.4</strong> failed. The build has been cancelled.

https://drone.owncloud.com/owncloud/core/32178/46/1

**Review by saw-jan — COMMENTED** [review]
LGTM

File: tests/acceptance/features/lib/ExternalStoragePage.php [hunk 1]
```diff
@@ -0,0 +1,89 @@
...
+
+	/**
+	 * {@inheritDoc}
+	 *
+	 * @return void
+	 * @throws Exception
+	 * @see \Page\FilesPageBasic::getFilePathInRowXpath()
+	 *
+	 */
+	protected function getFilePathInRowXpath():void {
```
Comment (Talank) [inline tests/acceptance/features/lib/ExternalStoragePage.php:86]: Why everything is protected in this class? 🤔

**Review by sakshamgurung — COMMENTED** [review]
Good good

File: tests/acceptance/features/lib/ExternalStoragePage.php (hunk 1 above)
Comment (SwikritiT) [reply tests/acceptance/features/lib/ExternalStoragePage.php:86]: Those methods are overridden from `FilesPageBasicPage` , that's why

**Review by swoichha — COMMENTED** [review]
LGTM

**Review by dpakach — COMMENTED** [review]
lgtm 👍🏼

**Review by kiranparajuli589 — COMMENTED** [review]
lgtm 👍🏼

**Review by phil-davis — APPROVED** [review]

**Pull request merged**